### DISCUSSION TITLE
mnode无法删除，

### DISCUSSION THREAD

**Discussion by XiaoFanQie2014** [taosdata/TDengine#28549] [Q&A]
docker images版本：3.3.2.0；
两个节点形成的集群，在taos命令行页面执行create mnode on dnode 2;没有报错，但是新的mnode一直处于creating状态，无法删除，第二个节点也无法删除，如下图：
![微信截图_20241030092334](https://github.com/user-attachments/assets/da335999-9fbe-4df5-830c-e4de0e0cc468)
现在创建新的数据库也报错，如下图：
![微信截图_20241030092318](https://github.com/user-attachments/assets/111ad19b-0ae8-4511-9c2e-59b9b89eb86f)
请问该怎么解决，谢谢！

**Comment by yu285** [comment]
遇到这个问题，先执行show transactions\G 看看与哪类事务有冲突。
